### ISSUE TITLE
[Programmatic access - Make Code - Homepage] Name and role are not defined for multiple tiles under Science section.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2361]
**User Experience:**
Screen reader user will find difficulty to identify the control if name and role are not defined for that control as user will skip the functionality over such control and also they will proceed further if he/she doesn't find any context.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome
Screen reader: Narrator, JAWS, NVDA

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta#
2. Home page will open.
3. Navigate to the controls appearing under "Science, Tools, Turtle" section in Home page.
4. Navigate to various tile like Timing Gates, Soil Moisture etc. under Science section.
5. Verify name and role of various tile like Timing Gates, Soil Moisture etc. under Science section.

**Actual Result:**
Name and role are not defined for various tile like Timing Gates, Soil Moisture etc. under Science section of homepage and due to this, user is unable to get the context of such control.

**Expected Result:**
Name and role should be defined for various tile like Timing Gates, Soil Moisture etc. under Science section of homepage so that screen reader user could get the context of such control and also could be able to perform functionality as well.

**Note:**
1. This issue exists throughout the tile appearing under Homepage. 
2. This issue also exists with Chrome + JAWS.
3. This issue also exists with Chrome + NVDA.

**MAS Reference:**
[MAS 4.1.2 - Name, Role, Value](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b248054a6-5e68-4771-9e1e-242fb5025730%7d)

[[Accessibility Insights!]](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

![MAS4 1 2_Homepage_Science](https://user-images.githubusercontent.com/30456910/62611990-42ab6200-b924-11e9-8add-81258878c1d8.png)
